### PR TITLE
conda-smithy v3.7.0

### PR DESCRIPTION
**Pull request by beckermr**
Merge only after success.

This pull request was auto-generated by [rever](https://regro.github.io/rever-docs/)

### CODE REVIEW

File: .scripts/build_steps.sh [hunk 1]
```diff
@@ -33,7 +34,7 @@ conda build "${RECIPE_ROOT}" -m "${CI_SUPPORT}/${CONFIG}.yaml" \
     --clobber-file "${CI_SUPPORT}/clobber_${CONFIG}.yaml"
 
 if [[ "${UPLOAD_PACKAGES}" != "False" ]]; then
-    upload_package "${FEEDSTOCK_ROOT}" "${RECIPE_ROOT}" "${CONFIG_FILE}"
+    upload_package  "${FEEDSTOCK_ROOT}" "${RECIPE_ROOT}" "${CONFIG_FILE}"
```
Comment (isuruf) [inline .scripts/build_steps.sh:37]: Why a space here?

Comment (beckermr) [reply .scripts/build_steps.sh:37]: This is jinja2. It removes some extra code for the output validation stuff and appears to have left an extra space. Likely can be fixed by the right combination of minus signs and percent signs. Feel free to make a PR!

File: .scripts/build_steps.sh [hunk 2]
```diff
@@ -19,7 +19,8 @@ conda-build:
 
 CONDARC
 
-conda install --yes --quiet conda-forge-ci-setup=2 conda-build -c conda-forge
+conda install --yes --quiet conda-forge-ci-setup=3 conda-build pip -c conda-forge
+
```
Comment (beckermr) [inline .scripts/build_steps.sh:23]: This newline is jinja2 as well.

File: .gitattributes [hunk 3]
```diff
@@ -17,7 +17,7 @@ bld.bat text eol=crlf
 .gitattributes linguist-generated=true
 .gitignore linguist-generated=true
 .travis.yml linguist-generated=true
-.scripts linguist-generated=true
+.scripts/* linguist-generated=true
```
Comment (beckermr) [inline .gitattributes:20]: This needs to be pushed to smithy.

**Comment by beckermr** [comment]
Good to merge @isuruf? We can push a bugfix to smithy later. I suspect we will need more.

**Comment by isuruf** [comment]
Is this ready for general use now? If so, okay.

**Comment by beckermr** [comment]
Yup. It should be fine for general use. It ships packages w/o validation just fine and it has the right hooks to allow validation when the time comes.

**Pull request merged**